controllers/daemonset_handler.go

```diff
@@ -81,9 +80,3 @@ func (r *KataConfigOpenShiftReconciler) isMCAvailable() (bool, error) {
 
 	return true, nil
 }
-
-
-func (r *KataConfigOpenShiftReconciler) daemonSetDeployment() (ctrl.Result, error) {
```

Comment:
This part of the commit should be squashed with 2c3da8db78973bba3f1479d551857690b9b6b65a